src/ImageSharp/Metadata/Profiles/IPTC/IptcProfile.cs

```diff
@@ -0,0 +1,281 @@
+                {
+                    this.values.Add(value.DeepClone());
+                }
+            }
+
+            if (other.Data != null)
+            {
+                this.Data = new byte[other.Data.Length];
+                other.Data.AsSpan().CopyTo(this.Data);
+            }
+        }
+
+        /// <summary>
+        /// Gets the byte data of the IPTC profile.
+        /// </summary>
+        public byte[] Data { get; private set; }
+
+        /// <summary>
+        /// Gets the values of this iptc profile.
+        /// </summary>
+        public IEnumerable<IptcValue> Values
+        {
+            get
+            {
+                this.Initialize();
+                return this.values;
+            }
+        }
+
+        /// <inheritdoc/>
+        public IptcProfile DeepClone() => new IptcProfile(this);
+
+        /// <summary>
+        /// Returns the value with the specified tag.
+        /// </summary>
+        /// <param name="tag">The tag of the iptc value.</param>
+        /// <returns>The value with the specified tag.</returns>
+        public IptcValue GetValue(IptcTag tag)
+        {
+            foreach (IptcValue iptcValue in this.Values)
+            {
+                if (iptcValue.Tag == tag)
+                {
+                    return iptcValue;
+                }
+            }
+
+            return null;
+        }
+
+        /// <summary>
+        /// Removes the value with the specified tag.
+        /// </summary>
+        /// <param name="tag">The tag of the iptc value.</param>
+        /// <returns>True when the value was found and removed.</returns>
+        public bool RemoveValue(IptcTag tag)
+        {
+            this.Initialize();
+
+            for (int i = 0; i < this.values.Count; i++)
+            {
+                if (this.values[i].Tag == tag)
+                {
+                    this.values.RemoveAt(i);
+                    return true;
+                }
+            }
+
+            return false;
+        }
+
+        /// <summary>
+        /// Changes the encoding for all the values.
+        /// </summary>
+        /// <param name="encoding">The encoding to use when storing the bytes.</param>
+        public void SetEncoding(Encoding encoding)
+        {
+            Guard.NotNull(encoding, nameof(encoding));
+
+            foreach (IptcValue value in this.Values)
+            {
+                value.Encoding = encoding;
+            }
+        }
+
+        /// <summary>
+        /// Sets the value of the specified tag.
+        /// </summary>
+        /// <param name="tag">The tag of the iptc value.</param>
+        /// <param name="encoding">The encoding to use when storing the bytes.</param>
+        /// <param name="value">The value.</param>
+        public void SetValue(IptcTag tag, Encoding encoding, string value)
+        {
+            Guard.NotNull(encoding, nameof(encoding));
+
+            if (!this.IsRepeatable(tag))
```

Comment:
@dlemstra i have changed the original code, so it is now possible to add some entrys more than once. If you think that would be useful for Magick.Net, too, i can change that over there, once this PR is done.

Reply:
Would be awesome if you could also get this merged into Magick.NET. Make sure you reference this issue then: https://github.com/dlemstra/Magick.NET/issues/262. And maybe the API then also needs this: `public IEnumerable<IptcValue> GetValues(IptcTag tag)`?